napari_animation/_qt/animation_widget.py

```diff
@@ -40,6 +41,9 @@ def __init__(self, viewer: 'napari.viewer.Viewer', parent=None):
         # Update theme
         self._update_theme()
 
+        # state interpolation updated needed because key-frame changed
+        self.up_to_date_interpolation = False
```

Comment:
might be clearer if this slider was an `AnimationSliderWidget` and this flag variable was a property of that object with a name like `is_up_to_date`

Reply:
`requires_update` maybe?

Reply:
Agreed, refactoring might be nice/ but we can also do it later after merge. Adding a tooltip / label would probably be helpful too